cpp/include/cuml/cluster/hdbscan.hpp

```diff
@@ -485,4 +471,22 @@ void out_of_sample_predict(const raft::handle_t& handle,
                            int min_samples,
                            int* out_labels,
                            float* out_probabilities);
+
+void _compute_core_dists(const raft::handle_t& handle,
```

Comment:
Since these new functions are exposed publicly, can you add some doxygen docs? I know there are few functions in here which also need docs. We should also go back and document those at some point as well.